include/nitro/registers_shared.h

```diff
@@ -10,8 +10,19 @@
 
 #include "nitro/types.h"
 
+#define reg_OS_TM0CNT_L            (*(REGType16v *)0x4000100)
```

Comment:
please ensure that these are removed from arm9/registers if they are in there, duplication probably isn't the best idea again